Cargo.toml

```diff
@@ -1,2 +1,3 @@
 [workspace]
-members = ["crates/*"]
\ No newline at end of file
+members = ["crates/*"]
+exclude = ["crates/cargo-bsp/tmp_test_project"]
```

Comment:
Ta zmiana była wymagana, ponieważ tworząc nowy Package wewnątrz innego projektu Rustowego, Cargo rozumie go jako część workspace. To było najprostsze rozwiązanie, ale można alternatywnie:

- sprawdzić uruchamianie testów w osobnych środowiskach
- zdecydować się na stworzenie 2 pakietów w mockowym projekcie już jako worskpace. Tworząc przy tym więcej Cargo.toml, a co za tym idzie więcej pracy z tym